### ISSUE TITLE
Syntax for loading trained model for re-training / fine tuning

### ISSUE THREAD

**Issue by shahlab-ucla** [CSBDeep/CSBDeep#64]
This wasn't entirely clear to me from the docs and example notebooks so I thought I'd ask. 

My understanding is that during training a new model is initialized by calling:
`model = CARE(config, 'my_model', basedir='models')`
With a config specified and that supplying config=none loads the trained model for prediction.

If the model already exists, does loading with a config and the model name specified load it with the pre-trained weights or does it initialize a new model?

In other words, what's the right syntax for loading a pre-trained model for further training / fine-tuning?

Thanks!

**Comment by uschmidt83** [comment]
Hi @shahlab-ucla,

there's more than one way to do this, but I'd do it like that:

1. Make a copy of the pre-trained model that I want to finetune further. 
2. Open the model via `model = CARE(None, 'my_tuned_model', basedir='models')`, thus loading the config of the pretrained model.
3. Modify the config for finetuning, e.g. `model.config.train_learning_rate = 1e-5`. (Also see [this example](https://forum.image.sc/t/how-to-continue-training/73482/5)).
4. Start training.
